src/controllers/activity/activity.ts

```diff
@@ -506,6 +510,15 @@ export class ActivityController extends EventEmitter {
     this.emitUpdate()
   }
 
+  hideBanner({ addr, network, timestamp }: { addr: string; network: string; timestamp: number }) {
+    this.#hiddenActivityBanners.push(`${addr}-${network}-${timestamp}`)
+
+    /* eslint-disable @typescript-eslint/no-floating-promises */
+    this.#storage.set('hiddenActivityBanners', this.#hiddenActivityBanners)
```

Comment:
How long should these be stored for? Shouldn't there be a cleaning mechanism?

Reply:
Isn't it a bit overkill to implement a cleaning mechanism for this? It's not like you're going to rage hit "close" on every single banner, not to mention that most of the time you will not be able to see it them at all as the txn will finish before actually seeing the activity

Reply:
Decided offline: to change the logic completely